registration-processor/registration-processor-message-sender-impl/src/main/java/io/mosip/registration/processor/message/sender/service/impl/MessageNotificationServiceImpl.java

```diff
@@ -128,6 +127,9 @@
 	/** The rest client service. */
 	@Autowired
 	private RegistrationProcessorRestClientService<Object> restClientService;
+	
+	@Value("#{'${mosip.default.template-languages}'.split(',')}")
```

Comment:
what if mosip.default.template-languages is not set, ie its null. Will this code fail. Since its not mandatory i think it will be better to use split after null check in line no- 281

Reply:
I have changed the code.